.github/workflows/build.yml

```diff
@@ -1033,6 +1033,37 @@ jobs:
                   path: python/perspective/dist/*.tar.gz
               if: ${{ matrix.os == 'macos-11' }}
 
+            ###############################
+            # Upload artifacts to release #
+            ###############################
+
+            - name: Upload dist artifact for Windows
+              if: ${{ runner.os == 'Windows' }}
+              uses: softprops/action-gh-release@v1
+              with:
+                  files: |
+                      python/perspective/dist/*.whl
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+            - name: Upload wheelhouse artifact for non-Windows
+              if: ${{ runner.os != 'Windows' }}
+              uses: softprops/action-gh-release@v1
+              with:
+                  files: |
+                      python/perspective/wheelhouse/*.whl
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+            - name: Upload sdist artifact for MacOS
+              if: ${{ matrix.os == 'macos-11' }}
+              uses: softprops/action-gh-release@v1
+              with:
+                  files: |
+                      python/perspective/dist/*.tar.gz
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
This isn't the canonical publish artifact for these (pypi), and we don't use them in any workflows. In that sense, this change  exclusively creates more work; it's a thing that can break that we don't use or have tests for, create confusion or distrust (if we break this in the future without noticing and publish different versions to pypi and GH releases).

It also can't be said to be more consistent, as we're not releasing _all_ of the artifacts, just the Python wheels.

We _could_ however use these for publishing, which would guarantee the artifacts are the same _and_ preserve the originals (since GitHub Actions expire eventually). To do this, you'd need to:

1) Update [publish.mjs](https://github.com/finos/perspective/blob/master/tools/perspective-scripts/publish.mjs) to download artifacts from a release instead of a build.
2) Upload all artifacts (including NPM artifacts) to release.